polyfill/rollup.config.js

```diff
@@ -27,7 +27,7 @@ export default [
   !isTest262 && {
     input: 'lib/index.mjs',
     plugins: [
-      replace({ ...replaceConfig, __debug__: !isProduction }),
+      replace({ ...replaceConfig, __debug__: !isProduction, preventAssignment: true }),
```

Comment:
A couple of suggestions for the rollup config file:
- It would be less repetitive to put `preventAssignment` directly in `replaceConfig`
- We could remove the non-test262, non-playground builds now altogether, I think?

Reply:
> * We could remove the non-test262, non-playground builds now altogether, I think?

OK, so remove the first section (that uses index.js as entry point) but leave the others?

> * It would be less repetitive to put `preventAssignment` directly in `replaceConfig`

Good idea. I'll make this change too.

Reply:
Right. (I wonder if that means we should remove `index.js` as well...)

Reply:
I don't have an opinion either way. Should I remove it?

Reply:
Yes, let's do it — I guess anyone who needed it should be using a different polyfill from now on.

Reply:
OK done!  Let's release and deprecate!